camera_system/game_motion_converter.py

```diff
@@ -285,3 +308,29 @@ def __get_next_edge(self, angle: int, current_edge: str) -> str:
                 next_edge = "left"  # 左側に回頭する場合エッジを左にする
 
         return next_edge
+
+    def __can_correction(self, coord: Coordinate, direct: Direction) -> bool:
```

Comment:
名前が変数っぽいし，何より実際の変数と名前が被ってるのは問題だと思う
setとかを付けた方がいいんじゃない
```suggestion
    def __set_can_correction(self, coord: Coordinate, direct: Direction) -> bool:
```